.github/workflows/percy_snapshots.yml

```diff
@@ -24,6 +24,8 @@ on:
         type: number
```

Comment:
(putting this comment on a random line)
I don't think this should stop us from using env variables, and I feel like we don't need to investigate right now if you don't know, but would someone technically be able to grab our mapbox apikey by making some code change (something like `alert(process.env.REACT_APP_MAPBOX_API_KEY)` in our storybook stories or a console.log in a node script), then running our CI?

Reply:
I popped open storybook just to check, and the mapbox api key is exposed in the get request for `https://api.mapbox.com/styles/v1/mapbox/streets-v11` so I'm wondering if there's a point of trying to hide our api key?

Reply:
Yeah you can print it on the page and it would show on the percy screenshot too if non approved users don't have access to view the ci